akka-actor/src/main/scala/akka/util/JavaDurationConverters.scala

```diff
@@ -3,13 +3,16 @@
  */
 package akka.util
 import java.time.{ Duration ⇒ JDuration }
-import scala.concurrent.duration.Duration
+
+import scala.concurrent.duration.{ Duration, FiniteDuration }
 /**
  * INTERNAL API
  */
 private[akka] object JavaDurationConverters {
+  def asScala(duration: JDuration): FiniteDuration = duration.asScala
```

Comment:
why the two styles? please remove this one

Reply:
I need call it in Java.